libraries-3/pom.xml

```diff
@@ -23,6 +23,11 @@
             <artifactId>lombok</artifactId>
             <version>${lombok.version}</version>
         </dependency>
+        <dependency>
+		        <groupId>com.github.rvesse</groupId>
```

Comment:
indent only with spaces

Reply:
Fixed. Also moved airline version to a specific parameter